libs/external-db-security/src/services/role_authorization.ts

```diff
@@ -1,41 +1,44 @@
+import { CollectionPermissions, VeloRole, WixDataRole } from "@wix-velo/velo-external-db-types"
```

Comment:
can you change it to single quotes? :innocent:

Reply:
of course, btw, don't we have a lint rule for it?